src/test/scala/de/upb/cs/swt/delphi/instanceregistry/connection/ServerTest.scala

```diff
@@ -23,27 +23,27 @@ import akka.http.scaladsl.model._
 import akka.http.scaladsl.server.Route
```

Comment:
I agree with your changes here. Please remove the commented out code completely :+1: Also, after addressing the comment above about ```GET /instances``` without any ```ComponentType``` specified, please add a test case for that.